docs/k0sctl-install.md

```diff
@@ -98,16 +111,17 @@ INFO Tip: To access the cluster you can now fetch the admin kubeconfig using:
 INFO      k0sctl kubeconfig              
 ```
 
-And -- presto! Your k0s cluster is up and running.
+### 4. Access the cluster
 
-#### 4. Access the cluster
+You must have the `kubeconfig` to access your k0s cluster.
```

Comment:
Sounds a bit blunt, considering the section is about how you can do exactly that. Now it sounds like a prerequisite.

```suggestion
You can use k0sctl to generate a `kubeconfig` for accessing your k0s cluster.
```

Reply:
Editing to read: "To access your k0s cluster, use k0sctl to generate a `kubeconfig` for the purpose."